spec/models/beard_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe 'Beard' do
+  before {
+    user1 = User.create(:first_name => "Paul", :last_name => "Jones")
```

Comment:
One thing you can improve here is lose the `before` and just do a: `let(:user) { User.create(:first_name => "Paul", :last_name => "Jones") }`

With this you can lose the `find_by_first_name` below.